test/Twig/Tests/FileExtensionEscapingStrategyTest.php

```diff
@@ -28,12 +28,13 @@ public function getGuessData()
             array('html', 'foo'),
             array('html', 'foo.bar.twig'),
             array('html', 'foo.txt/foo'),
-            array('html', 'foo.txt/foo.js/'),
```

Comment:
The problem is that `var_dump(pathinfo('foo.txt/foo.js/', PATHINFO_EXTENSION));` returns `string(2) "js"`. But this test is kinda strange anyway, as you cannot use a directory name here anyway. I don't see how this would be possible in twig.